recipes/root/all/conanfile.py

```diff
@@ -0,0 +1,210 @@
+import os
+import stat
+from contextlib import contextmanager
+from glob import glob
+from typing import List
+
+from conans import CMake, ConanFile, tools
+
+
+class PythonOption:
+    OFF = "off"
+    SYSTEM = "system"
+    # in future we may allow the user to specify a version when
+    # libPython is available in Conan Center Index.
+    ALL = [OFF, SYSTEM]
+    DEFAULT = OFF
+
+
+class RootConan(ConanFile):
+    name = "root"
+    version = "v6-22-02"
+    license = "LGPL-2.1-or-later"  # of ROOT itself, the recipe is under MIT license.
+    homepage = "https://root.cern/"
+    # ROOT itself is located at: https://github.com/root-project/root
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "CERN ROOT data analysis framework."
+    topics = ("data-analysis", "physics")
+    settings = ("os", "compiler", "build_type", "arch")
+    options = {
+        # Don't allow static build as it is not supported
+        # see: https://sft.its.cern.ch/jira/browse/ROOT-6446
+        # TODO: shared option should be reinstated when hooks issue is resolved
+        # (see: https://github.com/conan-io/hooks/issues/252)
+        # "shared": [True],
+        "fPIC": [True, False],
+        "python": PythonOption.ALL,
+    }
+    default_options = {
+        # "shared": True,
+        "fPIC": True,
+        "libxml2:shared": True,
+        "sqlite3:shared": True,
```

Comment:
I dont think we have any cases of setting defaults of the deps like this, I am not sure if this is allowed

Reply:
I will try removing these defaults.

Reply:
Removed in 7f503ea.